src/BodySummarizer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace GuzzleHttp;
+
+use Psr\Http\Message\MessageInterface;
+
+final class BodySummarizer implements BodySummarizerInterface
```

Comment:
```suggestion
final class TruncateBodySummarizer implements BodySummarizer
```

Just an idea!

Reply:
I went with `BodySummarizerInterface` rather than `BodySummarizer` because of the naming conventions of the other interfaces in this package.

Reply:
NB This class does more than just truncate. It also may throw away bodies that cannot be rendered easily.

Reply:
Well, I'd love to get rid of those interface suffixes 😄 

But I'm okay with delaying that to Guzzle 8 (if feasible at all)